examples/python/aoa.py

```diff
@@ -0,0 +1,98 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
+import jsbsim
+from IPython.core.display import display, HTML
+import xml.etree.ElementTree as ET
+import matplotlib.pyplot as plt
+import matplotlib.style as style
+style.use('fivethirtyeight')
+
+# function to change CG in aircraft xml
+# change the directory to the aircraft to be studied
+def changeCG(cgPos,readOnly):
+    tree = ET.parse('../../aircraft/global5000/global5000.xml')
```

Comment:
I don't feel comfortable with hardcoded relative paths because they assume the folder structure of the user is exactly the same than yours. At the very least, you should document what the directory structure is expected to be for this script to be run.